### ISSUE TITLE
[v2] Statistics for combined interfaces

### ISSUE THREAD

**Issue by RistiCore** [vergoh/vnstat#122]
Is it possible to get statistics for combined interfaces? In v1 it was possible to get it like: `vnstat -i tun0+tun1`?

**Comment by vergoh** [comment]
That's one feature that needed to be dropped for the version 2.0 release as the data structure was no longer compatible to make an easy merge with several interfaces. This change was mentioned in the CHANGES file:
```
 - Removed
   - Weekly output
   - Merge of data from multiple interfaces to one output
   - --testkernel
   - --exportdb, use --oneline, --json or --xml instead
   - --importdb, -u / --update and related options using vnstat command
- Old style (default in versions up to 1.7) summary layout in image output
```
This feature is however still something that may return one day if I can figure out some way of making the database engine do the work.

**Comment by vergoh** [comment]
Work has been started on a separate branch (https://github.com/vergoh/vnstat/tree/interface_merge) to restore this feature. Assuming there aren't any surprise blockers, this will get included in the version 2.2 release.
